spec/services/ccms/requestors/case_add_requestor_xml_blocks/non_passported_variable_attributes_spec.rb

```diff
@@ -318,6 +318,150 @@ module Requestors # rubocop:disable Metrics/ModuleLength
               end
             end
           end
+
+          context 'peps, unit trusts, bonds and gov. stocks' do
+            context 'applicant does not own peps, trusts, bonds or stocks' do
+              before { savings_amount.update! peps_unit_trusts_capital_bonds_gov_stocks: nil }
+              let(:attrs) do
+                %w[
+                  GB_INPUT_B_9WP2_4A
+                  GB_INPUT_B_9WP2_5A
+                ]
+              end
+              it 'does not generate the blocks' do
+                attrs.each do |attr_name|
+                  block = XmlExtractor.call(xml, :global_means, attr_name)
+                  expect(block).not_to be_present
+                end
+              end
+            end
+
+            context 'applicant does own peps, trusts, bonds or stocks' do
+              before { savings_amount.update! peps_unit_trusts_capital_bonds_gov_stocks: 54_321.0 }
+              let(:attrs) do
+                %w[
+                  GB_INPUT_B_9WP2_4A
+                  GB_INPUT_B_9WP2_5A
+                ]
+              end
+              it 'does not generate the blocks' do
+                attrs.each do |attr_name|
+                  block = XmlExtractor.call(xml, :global_means, attr_name)
+                  expect(block).to be_present
```

Comment:
You don't need to check for presence here - the next line `have_boolean_response` will do that for you